internal/app/dogma/get_cmd.go

```diff
@@ -264,6 +243,15 @@ func newGetCommand(c *cli.Context, out io.Writer) (Command, error) {
 		localFilePath = c.Args().Get(1)
 	}
 
+	if localFilePath == "/" || repo.path == "/**" {
+		localFilePath = repo.repoName
+	}
+
+	if localFilePath == "**" {
```

Comment:
How about using `else if` for this condition which seems mutually exclusive with the above condition.